xtask/src/check_todo.rs

```diff
@@ -51,7 +51,8 @@ impl Runnable for CheckTodo {
     fn run(self: Box<Self>, _opt: &Opt) -> Box<dyn Running> {
         let file_content = std::fs::read_to_string(&self.path).expect("couldn't read file");
         let invalid_todo_words = file_content
-            .split_whitespace()
+            .lines()
+            .map(|line| line.trim())
```

Comment:
Why didn't the tests catch this?

Reply:
Because we were splitting by word instead of by line (and the unit tests were operating on the output of that). That said, maybe we should reconsider. The reason why we cannot just run this on a real file is that it would cause false positives in CI (for the examples that don't match the correct format)